app/javascript/vue/shared/access_modal/edit.vue

```diff
@@ -31,10 +31,15 @@
         <div>
           <img :src="userAssignment.attributes.user.avatar_url" class="rounded-full w-8 h-8">
         </div>
-        <div>
-          <div>{{ userAssignment.attributes.user.name }}</div>
-          <div class="text-xs text-sn-grey">{{ userAssignment.attributes.inherit_message }}</div>
+        <div class="truncate"
+             :title="userAssignment.attributes.user.name
```

Comment:
Just fix on backend, and add current user flag.

Reply:
@aignatov-bio, It's used in API so we might break something!

Reply:
Different serializers for API and APP 
`app/serializers/api/v1/user_assignment_serializer.rb`
`app/serializers/user_assignment_serializer.rb`